uvm/js/common/ungrid/GridController.js

```diff
@@ -532,13 +532,13 @@ Ext.define('Ung.cmp.GridController', {
         return isEntryExists;
     },
 
-    checkBoxGroupStoreValidator:function(fieldConfig, valuesToCheck){
+    checkBoxGroupStoreValidator:function(isAlertEvent, fieldConfig, valuesToCheck){
         var availableFieldValues = [];
         if(fieldConfig.values && fieldConfig.values.length <= 0){
             return false;
         }else{
             for(var j=0; j<fieldConfig.values.length; j++){
-                availableFieldValues.push(fieldConfig.values[j][0]);
+                availableFieldValues.push(isAlertEvent ? fieldConfig.values[j] : fieldConfig.values[j][0]);
```

Comment:
Don't we have to do null check for `fieldConfig.values[j][0]` ?

Reply:
No when code reaches this point, it will definitely have values